docs/_docs/analysis.md

```diff
@@ -156,10 +157,41 @@ form factor of unity.
 `dq`        | _q_ spacing (1/Å)
 `com=true`  | Treat molecular mass centers as single point scatterers
 
+### Gyration Tensor
 
-### Radius of Gyration
+This calculates the gyration eigenvalues for a group of particles
+of same id.
 
-This calculate the radius of gyration, end-to-end distance, and related
+`gyration`   | Description
+---------------- | ----------------------------------------
+`nstep`          | Interval with which to sample
+`index`          | Particle id
```

Comment:
Is "index" a single value or can it be an array? Perhaps expand what "for a group of particles of same id" means and give formula for what is returned. Can results be save to disk at a certain interval like many of the other analysis?